schemas/blocks/calculated_summary.json

```diff
@@ -24,26 +24,48 @@
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/skip_conditions"
       },
       "calculation": {
-        "type": "object",
-        "properties": {
-          "calculation_type": {
-            "type": "string",
-            "enum": ["sum"]
-          },
-          "answers_to_calculate": {
-            "type": "array",
-            "minItems": 2,
-            "items": {
-              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+        "oneOf": [
+          {
+            "type": "object",
+            "properties": {
+              "calculation_type": {
+                "type": "string",
+                "enum": ["sum"]
+              },
+              "answers_to_calculate": {
+                "type": "array",
+                "minItems": 2,
+                "uniqueItems": true,
+                "items": {
+                  "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+                },
+                "description": "The id of the answers from which to do calculated_type to"
+              },
+              "title": {
+                "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
+              }
             },
-            "description": "The id of the answers from which to do calculated_type to"
+            "additionalProperties": false,
+            "required": ["calculation_type", "answers_to_calculate", "title"]
           },
-          "title": {
-            "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
+          {
+            "type": "object",
+            "properties": {
+              "operation": {
+                "type": "object",
+                "properties": {
+                  "$ref": "https://eq.ons.gov.uk/rules/operators/numeric/sum.json#/sum_answer_values"
+                },
+                "required": ["+"]
```

Comment:
This will need `"additionalProperties": false`

Reply:
As discussed needs to be resolved in 2020 json schema draft